client_plugin/vmdk_plugin/sanity_test.go

```diff
@@ -75,7 +76,7 @@ func init() {
 	flag.IntVar(&parallelVolumes, "parallel_volumes", 3, "Volumes per docker daemon for create/delete concurrent tests")
 	flag.IntVar(&parallelClones, "parallel_clones", 2, "Volumes per docker daemon for clone concurrent tests")
 	flag.Parse()
-	usingConfigFileDefaults := logInit(logLevel, logFile, configFile)
+	usingConfigFileDefaults := config.LogInit(logLevel, logFile, config.DefaultLogPath, configFile)
```

Comment:
Similar to `InitConfig(..)`, we could leave out the `config.DefaultLogPath` here.